features/nft/ntf-card/ntf-card.module.scss

```diff
@@ -1,6 +1,82 @@
+@import 'styles/typography';
+
 .root {
   border-radius: 8px;
   height: 424px;
-  object-fit: cover;
+  overflow: hidden;
+  position: relative;
   width: 296px;
 }
+
+.image {
+  border-radius: 8px;
+  object-fit: cover;
+}
+
+.preloader,
+.error {
+  align-items: center;
+  background-color: var(--color-primary-10);
+  border: 1px solid var(--color-neutral-50);
+  border-radius: 8px;
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  left: 0;
+  overflow: hidden;
+  position: absolute;
+  top: 0;
+  width: 100%;
+}
+
+.preloader {
+  background-color: var(--color-primary-10);
+}
+
+.error {
+  @extend %subtitle5;
+  border-color: var(--color-error-50);
+  color: var(--color-error-50);
+}
+
+.preloaderIcon {
+  display: inline-block;
+  height: 80px;
+  position: relative;
+  width: 80px;
+}
+
+.preloaderIcon div {
+  animation: preloader 1.2s cubic-bezier(0, 0.5, 0.5, 1) infinite;
+  background: var(--color-brand-purple-light);
+  display: inline-block;
+  left: 8px;
+  position: absolute;
+  width: 16px;
+}
+
+.preloaderIcon div:nth-child(1) {
```

Comment:
I would suggest to avoid usage of tags and nth-child if we can make the same using classes.